packages/web-twig/tests/components-fixtures/headerDialogButton.twig

```diff
@@ -0,0 +1,6 @@
+<HeaderDialogButton onClick="console.log('Hello!')">My account</HeaderDialogButton>
```

Comment:
No, I made it mandatory. On the other hand, `href` in `*Link` components is optional—but I'd like to make it mandatory too because the `href` is why you typically use a link component 🤷🏻‍♂️.

Reply:
Agree. Make it mandatory too please.